lib/components/src/ActionBar/ActionBar.tsx

```diff
@@ -1,5 +1,4 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { FunctionComponent } from 'react';
```

Comment:
nit: my personal preference is to use the `FC` short-hand since that makes the lines shorter

Reply:
But with IDE this should not be too problematic I believe. I personally think shorthand is evil since it is not really reader friendly.

Reply:
This comment reminds me that typing in React is really not an easy job... Maybe not so interesting, but I think some component could be more restrict to its children with this generic type recipe: `type FCNoChildren<P> = FunctionComponent<{ children?: never } & P>;` for those component should not ever have a children (i.e. always inlined).

Reply:
I agree with @leoyli on:
> shorthand is evil since it is not really reader friendly

Especially for react newbie like me.

Reply:
I almost always prefer longer names but this is one of the few exceptions since `FC`s are all over the place. It helps avoid line wraps when using descriptive component and prop names. I'd use it with `React.FC` though, importing just `FC` does feel like a little too much.

With all that said, if the convention so far has been to import `FunctionComponent` it makes sense to continue doing that.

Reply:
Let's keep the import as `FunctionComponent`. If desired, we can create the shorthand locally in the module.

This will allow newcomers to better understand what's going on, and find the proper documentation more easily.